src/assets/quickorder-sample/quickorder.csv

```diff
@@ -0,0 +1,3 @@
+Product ID,Qty
+201807231-01,2
+201807204,1
```

Comment:
Please rename the folder from `src/assets/quickorder-sample/` to `src/assets/sample-data/`